src/js/azdata/azdataQueryPlan.js

```diff
@@ -363,6 +364,39 @@ azdataQueryPlan.prototype.init = function (container, iconPaths, badgeIconPaths)
         return false;
     };
 
+    // Defines condition for showing the folding icon
+    graph.isCellFoldable = function (cell) {
```

Comment:
Can we please also add a configuration option when initializing a graph to show or hide the fold icon? We can disable this feature for plan comparison until we fix the polygon bug.